core/src/main/java/tc/oc/pgm/action/SoundType.java

```diff
@@ -0,0 +1,56 @@
+package tc.oc.pgm.action;
+
+public enum SoundType {
+  CUSTOM("note.pling", 1f, 1f),
+  RAINDROP("random.levelup", 1f, 1.5f),
+  TIP("mob.endermen.idle", 1f, 1.2f),
+  ALERT("note.pling", 1f, 2f),
+  CLICK("random.click", 1f, 2),
+  PORTAL("mob.endermen.portal", 1f, 1f),
+  SCORE("random.levelup", 1f, 1f),
+  PURCHASE("fire.ignite", 1f, 1.4f),
+  DEFUSE("random.fizz", 1f, 1f),
+  OBJECTIVE_FIREWORKS_FAR("fireworks.blast_far", 0.75f, 1f),
+  OBJECTIVE_FIREWORKS_TWINKLE("fireworks.twinkle_far", 0.75f, 1f),
+  OBJECTIVE_COMPLETION_ALLY("portal.travel", 0.7f, 2f),
+  OBJECTIVE_COMPLETION_ENEMY("mob.blaze.death", 0.8f, 0.8f),
+  OBJECTIVE_MODE("mob.zombie.remedy", 0.15f, 1.2f),
+  CONTROL_POINT_CAPTURE("portal.travel", 0.35f, 2f),
+  CONTROL_POINT_LOSE("mob.blaze.death", 0.4f, 0.8f),
+  FLAG_PICKUP_OWN("mob.wither.idle", 0.7f, 1.2f),
+  FLAG_DROP_OWN("mob.wither.hurt", 0.7f, 1f),
+  FLAG_RETURN_OWN("mob.zombie.infect", 1.1f, 1.2f),
+  FLAG_PICKUP("entity.firework_rocket.blast_far", 1f, 0.7f),
+  FLAG_DROP("entity.firework_rocket.twinkle_far", 1f, 1f),
+  FLAG_RETURN("entity.firework_rocket.twinkle_far", 1f, 1f),
```

Comment:
i really wonder if we *really* need *all* of these
i think it's good to give a few (5-10) preset sounds and if they want smthg else they can use the resource key, but giving every single pgm sound (~30) as a preset seems overkill, because no one will know them all and listing the 30 sounds in documentation without proper explanation as to which each of them is, will be pain.

Reply:
Yeah I was mostly going for completion. It would be a pain to document all of these. I'll try and reduce this list.

Reply:
I've reduced it to 12